### PR TITLE
feat(app, components): add assets for absorbance reader and display them in app

### PR DESCRIPTION
**Pull request by smb2268**
# Overview
This initial absorbance reader PR adds the design assets, displays them where appropriate, removes the need for calibration and wires up the 

## Changelog
1. Add icon, SVG rendering and PNG image and wire them up
2. Wire up module card status to show whether reading or idle and whether the lid is on
3. Wire up module overflow menu to show one "about module" item
4. Remove calibration requirement for this module

## Review requests
Look through screenshots! You can test this out if you want also

Screen Shot 2024-10-09 at 3 09 43 PM
Screen Shot 2024-10-09 at 3 14 44 PM
Screen Shot 2024-10-09 at 3 10 31 PM
Screen Shot 2024-09-24 at 4 45 57 PM
Screen Shot 2024-09-24 at 2 31 10 PM
Screen Shot 2024-09-24 at 2 30 56 PM
Screen Shot 2024-09-24 at 1 04 47 PM
Screen Shot 2024-09-24 at 1 04 33 PM

## Risk assessment
Low

### CODE REVIEW

**Review by vegano1 — APPROVED** [review]
Very nice!
Looks good to merge once CI passes.

File: components/src/hardware-sim/Module/PlateReader.tsx [hunk 1]
```diff
@@ -0,0 +1,210 @@
+import * as React from 'react'
```
Comment (mjhuff) [inline components/src/hardware-sim/Module/PlateReader.tsx]: We don't need to do this anymore with the recent lint changes 👍

File: app/src/organisms/Devices/ProtocolRun/SetupModuleAndDeck/utils.ts
Comment (mjhuff) [inline app/src/organisms/Devices/ProtocolRun/SetupModuleAndDeck/utils.ts:1]: Nice

**Review by mjhuff — APPROVED** [review]
lgtm!

**Pull request merged**